doc/nrf/ug_thingy91_gsg.rst

```diff
@@ -11,6 +11,11 @@ Getting started with Thingy:91
 This guide helps you get started with Thingy:91.
 It tells you how to update the Thingy:91 application and modem firmware and connect the Thingy:91 to `nRF Cloud`_.
 
+If you have already set up your Thingy:91 and want to learn more:
+
+* If you are already familiar with the |NCS|, see :ref:`ug_thingy91` for more advanced topics related to the Thingy:91.
+* If you are not familiar with the |NCS| and the development environment, see the :ref:`introductory documentation <getting_started>`.
```

Comment:
```suggestion
If you have already set up your Thingy:91 and want to learn more, see the following documentation:

* :ref:`introductory documentation <getting_started>` for more information on |NCS| and the development environment.
* :ref:`ug_thingy91` for more advanced topics related to the nRF9160 DK if you are already familiar with the |NCS|.
```

Reply:
I like the rewording. Not sure which order is better.

I like that recommending the NCS comes first since that's what you should know before going for the Developing with pages. But on the other hand, the Developing with page follows naturally from the Getting started with page, in which case it should be first.

Reply:
I can see that we are referring to getting started from the building and programming section of Developing with Thingy guide: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91.html#building-and-programming-from-the-source-code for setting up the system. I don't see it in the DK guide and we are linking directly to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_programming.html#gs-programming. May be we need to link to getting started before the link to the programming section? And since the user needs to have the set up ready before building and programming from the source code (which is one of the sections in 'developing with' guides, I think the suggested order is ok.